samples/net/rpl-node/0001-Added-CoAP-support-for-Sparrow-Border-Router.patch

```diff
@@ -0,0 +1,272 @@
+From cb38f06f84e8d3c1e021265d40a21bc16b5a8a44 Mon Sep 17 00:00:00 2001
```

Comment:
Is this file supposed to be here?

Reply:
Nix that.  I noticed you're keeping a border router patch in the sources.  I forgot to remove this comment.